guides/v2.1/javascript-dev-guide/widgets/widget_tabs.md

```diff
@@ -244,6 +244,36 @@ Called before the content is opened.
 ### `dimensionsChanged` {#fedg_tabs_dimensionsChanged}
 Called after content is opened or closed.
 
+## Code Sample {#tabs_code_sample}
```

Comment:
Please, remove `{#tabs_code_sample}` part. As it's preferable to use a clean Markdown, the links to the sections can be processed using Markdown.